docs/content/preview/integrations/cdc/debezium.md

```diff
@@ -2,7 +2,7 @@
 title: Debezium and CDC in YugabyteDB
 linkTitle: Debezium
 description: Using Debezium for CDC in YugabyteDB.
-earlyAccess: /preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+earlyAccess: /preview/releases/versioning/#feature-availability
```

Comment:
why not link it to Earky access instaed? - /preview/releases/versioning/#early-access-ea

Reply:
Thought about it, but felt it was useful to give all the context.

Reply:
yes fair point and actually easy maintaining it across too!